pages/SourceCode/RetailConsoleSourceCode.md

```diff
@@ -37,9 +37,6 @@ Nintendo systems have by far the largest selection of official source code to ch
 ### Days of Thunder 
 {% include link-to-other-site.html url="https://github.com/DickBlackshack/Days-of-Thunder-NES-Unpublished" description="Days of Thunder was an unpublished NES game by developer Chris Oberth" image="https://images-na.ssl-images-amazon.com/images/I/61rhbiNhkoL._AC_.jpg" title="Days of Thunder"  %}
 
-### Infinity
```

Comment:
Why was Infinity removed?

Reply:
Linked repo was deleted or moved. I found [what might be where it moved](https://github.com/gb-archive/infinity-gbc), but I'm not 100% sure if it's the same content. Adding it back would be easy once it's confirmed.